ansible/README.md

```diff
@@ -37,7 +37,7 @@ Once you have the variables set, you can...
 Execute the command from your local machine:
 
 ```bash
-ansible-playbook -i host deploy_regular_node.yml -vv
+ansible-playbook -i host deploy_regular_node_docker.yml -vv
```

Comment:
Both deployments will live, so we need to differentiate in the README, or maybe having a docs folder with the differences and link both documents to this main one (Docker and non Docker with Ansible), whatever you think best.